tmp

```diff
@@ -0,0 +1,374 @@
+rsync --archive --exclude='*.jsx' node_modules/backbone/backbone.js built/add-on/chrome/content/shared/vendor/backbone.js
```

Comment:
If you didn't intend to add this file, you should be aware of what your git commit commands are doing.